src/desktop/apps/article/routes.js

```diff
@@ -37,13 +37,17 @@ export async function index(req, res, next) {
     })
     const article = data.article
     const articleModel = new Article(data.article)
+    const utm =
```

Comment:
Since UTM is just a name for a certain type of parameter, I think it makes more sense to call this `querystring` or `search` -- also note that there are lots of ways to parse urls like [this](https://nodejs.org/api/url.html#url_url_search) which feels a little more standard.

Reply:
Oh cool, URL looks like a better option -- couldn't get a direct route to it through the express `req`